test/Elastic.Apm.AspNetFullFramework.Tests/TestsBase.cs

```diff
@@ -143,7 +143,7 @@ internal static class SampleAppUrlPaths
 				new SampleAppUrlPathData("Asmx/Health.asmx", (int)HttpStatusCode.OK);
 
 			internal static readonly SampleAppUrlPathData ChildHttpSpanWithResponseForbiddenPage =
-				new SampleAppUrlPathData(HomeController.ChildHttpSpanWithResponseForbiddenPath, 200, spansCount: 1, errorsCount: 0);
+				new SampleAppUrlPathData(HomeController.ChildHttpSpanWithResponseForbiddenPath, 200, transactionsCount:2, spansCount: 1, errorsCount: 0);
```

Comment:
Since we call the same service in `ChildHttpSpanWithResponseForbiddenPath`, we expect 2 transactions (and not 1 as before).